dart/colfer_gen.dart

```diff
@@ -0,0 +1,831 @@
+// Code generated by colf(1); DO NOT EDIT.
+// The compiler used schema file test.colf for package gen.
+// Package gen tests all field mapping options.
+
+import 'dart:typed_data';
+import 'package:collection/collection.dart';
+import 'package:quiver/core.dart';
+import 'dart:convert';
+
+/// The upper limit for serial byte sizes.
+final colferSizeMax = 16 * 1024 * 1024;
+
+/// The upper limit for the number of elements in a list.
+final colferListMax = 64 * 1024;
+
+void encodeVarint(BytesBuilder buf, int x) {
+  while (x > 127) {
+    buf.addByte((x & 127) | 128);
+    x >>= 7;
+  }
+  buf.addByte(x);
+}
+
+int decodeInt64(Uint8List data, int i) {
```

Comment:
I think Dart already provides this functionality. We'll benefit from more efficiency too.

https://api.dart.dev/stable/2.13.0/dart-typed_data/UnmodifiableByteBufferView/asInt64List.html

Reply:
Nice catch!